lib/board.rb

```diff
@@ -0,0 +1,67 @@
+class Board
+	
+	attr_accessor :cells
+	
+
+	
+	def reset! 			
+		cells.clear
+		self.cells = Array.new(9, " ")
+	
+	end
+
+	def initialize		
+		@cells = Array.new(9, " ")
+	end
+
+	def display
+		puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+  		puts "-----------"
+  		puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+  		puts "-----------"
+  		puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+	end
+
+	def position(val)
+		cells[val.to_i - 1]
+
+	end
+
+	def full?
+		if cells.all?{|full| full == "X" || full == "O"}
```

Comment:
Using `all?` will return either a true or false value by default so no need to use an if else condition. 

`cells.all?{|full| full == "X" || full == "O"}`